Lorem Ipsum XR Translator/Assets/Scripts/TestBehaviors/ObjectDetectBehavior.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.Networking;
+using TMPro;
+using ObjectDetection;
+using Config;
+using UnityEngine.XR.ARSubsystems;
+using System.Collections.Generic;
+
+public class ObjectDetectBehavior : MonoBehaviour
```

Comment:
Do we still need this?

Reply:
No, the raycaster I was testing there was also implemented in the sceneanalyzer, so this can go as well.